changes/12888-apple-rapid-security-response

```diff
@@ -0,0 +1 @@
+Added Apple Rapid Security Response version to host details.
```

Comment:
Let's document that this feature (new `extra` column on `os_version` for macOS devices) was added to osquery in v5.9.1. So macOS devices will need >= v5.9.1 version installed.

Reply:
@lucasmrod I added this detail to changes/12888-apple-rapid-security-response. Please re-approve this pull request.